src/App.js

```diff
@@ -19,6 +19,7 @@ function App() {
 					</Route>
 					<Route path='/AddItem'>
 						<AddItem />
+            {/* <AddItem token={token} /> */}
 					</Route>
           <Route path='/FirebaseTesting'>
 						<FirebaseTesting />
```

Comment:
As you continue to build out the app, you all (@dkimlim, @AshlingH , @jbatara , @jonbascos ) might want to look into using `createContext` and `useContext` to share state.
Create Context:  https://reactjs.org/docs/context.html
useContext hook: https://reactjs.org/docs/hooks-reference.html#usecontext